include/xrpl/protocol/XRPAmount.h

```diff
@@ -160,6 +158,11 @@ class XRPAmount : private boost::totally_ordered<XRPAmount>,
         return drops_ != 0;
     }
 
+    operator Number() const noexcept
```

Comment:
Would `explicit` be useful here ?

Reply:
I will try that.

Reply:
There are too many call sites depending on the implicit conversion.